tests/translator/output/error_api_swagger_integration_with_getatt_intrinsic_api_id.json

```diff
@@ -1,3 +1,3 @@
 {
-  "errorMessage": "Invalid Serverless Application Specification document. Number of errors found: 1. Resource with id [HtmlFunction] is invalid. Event with id [GetHtml] is invalid. Api Event's RestApiId must be a string referencing an Api in the same template."
+  "errorMessage": "Invalid Serverless Application Specification document. Number of errors found: 1. Resource with id [HtmlFunction] is invalid. Event with id [GetHtml] is invalid. Property 'RestApiId' should be a string."
```

Comment:
The "in the same template" is not checked here, we have another test to cover it: https://github.com/aws/serverless-application-model/blob/develop/tests/translator/output/error_http_api_event_invalid_api.json